client/templates/course/sidebar/sidebar.less

```diff
@@ -0,0 +1,16 @@
+.course-left-nav ul > li a {
+    padding: 1em;
+}
+
+.course-left-nav ul > li a:hover {
```

Comment:
Use semantic class names, such as 'primary navigation' or 'course-navigation'. Conflating class names with positional or style elements, such as 'left' or 'big', is [considered non-semantic](https://css-tricks.com/semantic-class-names/).